packages/mwp-app-render/src/components/uxcapture/uxCaptureConfig.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import UXCaptureConfig from './UXCaptureConfig';
+
+const MOCK_PROPS = {
+	onMark: foo => {
+		if (foo) {
+			return foo;
+		}
+	},
+	onMeasure: (bar, noop) => {
```

Comment:
I think `onMeasure` should only get one argument?

Reply:
yeppp, not sure what happened here. :)